[NEW] new coop_print_badge module

jweste opened the pull request.

jweste committed:
[NEW] new coop_print_badge module

legalsylvain commented on louve_addons/coop_print_badge/report/report_printbadge.xml:
```diff
@@ -0,0 +1,37 @@
+                    </div>
+
+                    <div class="coop_text">
+                        COOPÉRATEUR
```
Yop ! 
Tu peux mettre en anglais, le fichier de traduction fait ensuite le taff, c'est plus homogène avec le reste du module en anglais.

legalsylvain reviewed (commented):
- petite remarque sur une traduction.
- pour l'icon, peut-être mettre des données et images anonymes, vu que c'est ensuite publique ? 

Sinon, c'est cool.

jweste committed:
module coop_membership_access_rights
[IMP] Create the view for group_membership_access_user users

jweste committed:
[FIX] put english word on report and translate it

jweste committed:
[FIX] change module icon

jweste commented:
corrections suite aux remarques de @legalsylvain

jweste committed:
module coop_print_badge:
[IMP] call a '/report /barcode/EAN13/xxxxxxx' url instead of computing the image.

jweste committed:
[i18n]

jweste committed:
module coop_print_badge:
[FIX] try to split member name if too long

jweste committed:
module coop_print_badge:
[IMP] print a different color wether partner is member or associated

jweste commented on louve_addons/coop_print_badge/static/src/css/badge.css:
```diff
@@ -0,0 +1,59 @@
+    width:65mm !important;
+    overflow: hidden;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
```
@legalsylvain Je n'arrive pas à faire en sorte que le texte ne dépasse pas le cadre si le partner a un nom trop long. C'est bien possible de le faire en css, non?
